File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/shop/CakeShopUpdateParam.java
package com.cakk.domain.mysql.dto.param.shop;

import lombok.Builder;

import com.cakk.domain.mysql.entity.user.User;

@Builder
public record CakeShopUpdateParam(
	String thumbnailUrl,
	String shopName,
	String shopBio,
	String shopDescription,
	User user,
	Long cakeShopId
) {
}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/shop/CakeShopUpdateParam.kt
package com.cakk.domain.mysql.dto.param.shop

import com.cakk.domain.mysql.entity.user.User
import lombok.Builder

data class CakeShopUpdateParam(
	val thumbnailUrl: String,
	val shopName: String,
	val shopBio: String,
	val shopDescription: String,
	val user: User,
	val cakeShopId: Long
)
